### PR TITLE
Update install-sqoop bootstrap script.

### PR DESCRIPTION
**Pull request by mtyaka**
The script was referencing old versions of sqoop and MySQL connector packages
that aren't available anymore.

*Partner Information*: not an edX partner - 3rd party-hosted open edX instance
*JIRA Ticket*: https://openedx.atlassian.net/browse/OSPR-677

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @mtyaka! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request. 

To automatically create an OSPR issue for this pull request, just visit this link: http://openedx-webhooks.herokuapp.com/github/process_pr?number=13&repo=edx%2Fedx-analytics-configuration

**Comment by edx-webhook** [comment]
Thanks for the pull request, @mtyaka! I've created [OSPR-677](https://openedx.atlassian.net/browse/OSPR-677) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams. 

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ('this must be merged by XX date', and why that is)
- partner information ('this is a course on edx.org')
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the Github pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

We can't start reviewing your pull request until you've added yourself to the [AUTHORS](https://github.com/edx/edx-analytics-configuration/blob/master/AUTHORS) file. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

File: batch/bootstrap/install-sqoop [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
-SQOOP_VERSION=1.4.5
+SQOOP_VERSION=1.4.6
 SQOOP_RELEASE=sqoop-$SQOOP_VERSION.bin__hadoop-$HADOOP_VERSION
 SQOOP_RELEASE_FILENAME=$SQOOP_RELEASE.tar.gz
 SQOOP_HOME=/usr/lib/sqoop
 SQOOP_LIB=$SQOOP_HOME/lib
 
-MYSQL_CONNECTOR_RELEASE=mysql-connector-java-5.1.29
+MYSQL_CONNECTOR_RELEASE=mysql-connector-java-5.1.35
```
Comment (mulby) [inline batch/bootstrap/install-sqoop]: Maybe we could make this a script parameter with a default value?

I could imagine people wanting to override this to put in whatever version they have handy.

Also, while you're in here changing stuff can you add a "set -e" at the top? We have had this script fail and EMR think that it succeeded...

**Comment by mulby** [comment]
Generally looks good, just a couple small tweaks suggested.

**Comment by mtyaka** [comment]
@mulby Thanks! I added `set -e` to the top and made the script accept optional `--hadoop-version`, `--sqoop-version`, and `--mysql-connector-version` parameters.

**Comment by mulby** [comment]
This generally looks good, I'm running the acceptance tests on a cluster provisioned with this script now.

**Comment by mulby** [comment]
Acceptance tests passed. Merging.

**Pull request merged**